src/main/common/filter.h

```diff
@@ -54,20 +56,32 @@ typedef struct firFilterDenoise_s {
 } firFilterDenoise_t;
 
 typedef struct fastKalman_s {
-    float q;       // process noise covariance
-    float r;       // measurement noise covariance
-    float p;       // estimation error covariance matrix
-    float k;       // kalman gain
+    float k;       // "kalman" gain
     float x;       // state
     float lastX;   // previous state
 } fastKalman_t;
 
+typedef struct laggedMovingAverage_s {
+    int movingWindowIndex;
```

Comment:
Smaller type may be worth the memory saving ...

Reply:
uint8_t should be fine.. not sure why I went with int on this one